sdk/cosmos/changelog/README.md

```diff
@@ -1,4 +1,8 @@
 ## Changelog
```

Comment:
Why isn't this file called CHANGELOG.md?

Reply:
Old V3 stuff

Reply:
We can update it later. Though we have tried not to change V3 structure at this point - just want to leave it as it is as we rarely interact with it.

Reply:
OK, I missed this was in the v3 branch.